generators/bdk/java/index.js

```diff
@@ -73,23 +73,14 @@ module.exports = class extends Generator {
         this.answers.host = this.options.host;
         this.answers.username = this.options.username;
 
-        // get latest BDK BOM version
```

Comment:
+1 on cleaning up dead code, you can also remove the constants at the top (MAVEN_BDK_BOM_SEARCH_BASE MAVEN_SPRING_BOOT_SEARCH_BASE)